### DISCUSSION TITLE
Terminando el curso se puede encontrar trabajo en alguna empresa?

### DISCUSSION THREAD

**Discussion by geraodst117** [Launch-X-Latam/MisionFrontEnd#70] [Q&A]
Pues esa es una duda que tengo, al terminar podríamos buscar trabajo como desarrollador?

**Comment by Cerecero** [comment]
Creo yo que dependera de cada uno, si te sientes capaz con los conocimientos que adquiriste durante el LaunchX si.

**Reply by EsthebanDiego** [reply]
Habían mencionado que era posible que te refirieran a empresas afiliadas, pero era un muy depende de como te valla en el curso, tampoco lo garantizaban

**Comment by JosafatJimenezB** [comment]
Al terminar ya tendras las bases como para profundizar un poco mas en lo que aprendas en este bootcamp, al finalizar si participas frecuentemente en las actividades te daran la oportunidad de entrar a innovaccion network para ver vacantes relacionadas con lo aprendido
pero no te garantizan que seas contratado.

**Comment by GustavoGutierrez21** [comment]
Normalmente si acabas todo llenas un formulario donde ellos tienen oportunidades laborales externas, pero solo son el contacto, ya depende de ti que la empresa te responda, pero si estarás ahí registrado.

**Comment by albertt206** [comment]
De algo es seguro! si eres persistente en el curso tendras los conocimientos suficientes para poder hacer muchas cosas incluyendo especializarte o encontrar un trabajo (aunq sea de practicante)

**Comment by koshishark** [comment]
Ellos no te lo garantizan. Pero, como todo en la vida, dependerá de qué hagas con la información que adquieres en este bootcamp. Si no la amplías y tratas de profundizar en ello, es casi seguro que no conseguirás el trabajo que quieres. Te recomiendo hacer tu portafolio de desarrollo (aquí mismo en GitHub) e ir realizando proyectos con lo que vayas aprendiendo. Llegará el punto donde tengas todo para aplicar por los trabajos (que sobrarán las ofertas, también te digo), e incluso, si eres consistente y aplicas bien lo que haces, las ofertas lleguen a ti sin buscarlas. Mucho éxito, paciencia y, sobre todo, constancia.

**Comment by FernandoPeraltaCarlos** [comment]
No, tomar un curso no te vuelve desarrollador , algunos hasta una ingeniería tenemos.

**Reply by GustavoLoreto** [reply]
Cierto, muy cierto!

**Comment by MaferMtz** [comment]
En mi experiencia con innovaccion virtual nos registraron a un portal de reclutamiento de la que he recibido unas cuantas ofertas como desarrollador pero yo ya me encontraba trabajando, igual depende de lo que tu busques y que sigas siempre preparandote y practicando para llenar los requerimientos que piden en la empresa y ademas como te sepas mover, los trabajos no llegan de la nada, si hay pero tiens que saber buscar

**Reply by NoeReyesGo** [reply]
Hola. Estuve en Innovaccion y pasé la certificación AZ-900, pero no me enteré del portal de reclutamiento ¿Cómo te enteraste tú?

**Comment by T0ny-dev** [comment]
Hola geraodst117, una duda que en algún momento tenemos todos, de mi circulo de cercanos te puedo decir que si, todas las personas somos distintos pero si fortaleces tus habilidad blandas , técnicas como con esta experiencia te puedo decir que lo conseguirás , cuento con una amiga arquitecta que entro desde cero y ahora trabaja en una start up de estados unidos , conocí un arquitecto del banco ripley que estuvo intentando lo 4 veces aprender a codear y ahora es todo un crack recuerda que puede ser la misma tarea y diferente experiencia por eso varia tanto pero si es posible , podrías aprender aparte de codear también a testear código  , CI/CD  e ir incrementando tu ingles , por mi parte sigo aprendiendo , conociendo personas en este mundo y te puedo decir que se valora mas la capacidad de poder resolver problemas que un curso o algo similar. asi que dale sin miedo 🚀

**Reply by Mauricio-Mizhquiri** [reply]
En donde estudias ingles, o que recursos usas?

**Reply by T0ny-dev** [reply]
hola bro , yo el ingles lo repase bastante con este recurso: https://www.youtube.com/watch?v=I_v4K8_jlnQ , tambien uso duolingo para agregar nuevas palabras pero ese curso de dave romero esta muy bueno , del code pues sigo  Midu , hola mundo , fazt code edutuber y mucho a jhonmircha , de los cursos pues estudie bastante en codeacademy , los de freecodecamp , udemy , no te recomiendo mucho crehana en aspectos de estudiar programación los cursos son muy abiertos  y cortos y no profundizan mucho , otros cursos ahí como diseño están del padrisimos los del code les falta mas trabajo

**Comment by samujm** [comment]
En el curso unicamente nos estan dando bases o herramientas , ya corre por nuestra cuenta investigar más y reforzarlas, en todo caso ser más curioso para todas esas "palabras claves" investigarlas más a fondo y conocer cada día un poco más. Animo!!!
